disnake/asset.py

```diff
@@ -118,6 +119,43 @@ async def save(
             with open(fp, "wb") as f:
                 return f.write(data)
 
+    async def to_file(self, *, spoiler: bool = False) -> File:
+        """|coro|
+
+        Converts the asset into a :class:`File` suitable for sending via
+        :meth:`abc.Messageable.send`.
+
+        .. versionadded:: 2.0
+
+        Parameters
+        -----------
+        spoiler: :class:`bool`
+            Whether the file is a spoiler.
+
+        Raises
+        ------
+        DiscordException
+            The asset does not have an associated state.
+        TypeError
+            The asset is a sticker with lottie type.
+        HTTPException
+            Downloading the asset failed.
+        Forbidden
+            You do not have permissions to access this asset
```

Comment:
```suggestion
```
I don't think there are any assets that could result in a 403; it would apply to `Attachment`, but that one (currently) doesn't inherit from `AssetMixin`.

Reply:
Yep, It seems like you're right. As far as I can tell, it doesn't look like Discord is currently restricting access to asset urls.